test/images/node-perf/tf-wide-deep/BASEIMAGE

```diff
@@ -1 +1 @@
-linux/amd64=python:3.6-slim-stretch
+linux/amd64=python:3.9-slim-bullseye
```

Comment:
any specific reason for sticking to 3.9, its support is ending soon(2025-10) per the [python website](https://devguide.python.org/versions/)